rhcephcompose/compose.py

```diff
@@ -46,6 +46,8 @@ def __init__(self, conf):
         self.product_version = conf['product_version']
         # Extra files to put at the root of the compose
         self.extra_files = conf['extra_files']
+        # Whether sources composition should be skipped
+        self.no_sources = conf['no_sources']
```

Comment:
That works. Do you think it's alright to have the inverse as the command line option still? `--no-sources` :arrow_right: `include_sources = False`?